### PR TITLE
OKD 4.16 and 4.17 release announcement and supporting documentation

### PR DESCRIPTION
**Pull request by GingerGeek**
Includes:
- Blog post for 4.17/4.16 release
- Upgrade notes for getting a 4.15 cluster to 4.16
- Supporting notes about FCOS -> SCOS

### CODE REVIEW

File: docs/project/upgrade-notes/from-4-15/fcos-to-scos-migration.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+:::note
+If you experience issues transitioning between FCOS and SCOS please report or start a discussion on our [GitHub project](https://github.com/okd-project/okd).
+:::
+
+We have seen reports related to ext4 features that were present on FCOS installs but not available on SCOS.
+
+### Will you reintroduce a method to support Fedora based node operating systems or base images
+
+There are lab, research and experimental reasons which may mean you want to be running newer packages or kernels than what's available in CentOS Stream. Should you have the resource to do so, we encourage contributions in this area to look at how we can provide both alternative Node Operating Systems (ie Fedora) and even base contianer images for certain cluster components. Please get in touch!
```
Comment (varesa) [inline docs/project/upgrade-notes/from-4-15/fcos-to-scos-migration.md:33]: "contianer" typo here

File: blog/2024/07-30-okd-pre-release-testing.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 draft: false 
 date: 2024-07-30
+authors: ["jaimemagiera"]
```
Comment (rassie) [inline blog/2024/07-30-okd-pre-release-testing.md:4]: Don't want to nitpick too much, but does this really belong in this patch?

File: docs/project/upgrade-notes/from-4-15/fcos-to-scos-migration.md (hunk 1 above)
Comment (GingerGeek) [reply docs/project/upgrade-notes/from-4-15/fcos-to-scos-migration.md]: Fixed thank you :)

File: blog/2024/07-30-okd-pre-release-testing.md (hunk 2 above)
Comment (GingerGeek) [reply blog/2024/07-30-okd-pre-release-testing.md:4]: It's a fair flag and you are right that strictly it should be in a separate changeset.

Given it's low-risk and a minor change I won't pull it out for now

**Pull request merged**